dev/AppLifecycle/AppActivationArguments.h

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+#include "Microsoft.Windows.AppLifecycle.AppActivationArguments.g.h"
+
+namespace winrt::Microsoft::Windows::AppLifecycle::implementation
+{
+    struct AppActivationArguments : AppActivationArgumentsT<AppActivationArguments>
+    {
+        AppActivationArguments(winrt::Windows::ApplicationModel::Activation::IActivatedEventArgs platformArgs)
```

Comment:
Take C++/WinRT types by `const&` so `IActivatedEventArgs const& platformArgs`

Reply:
Are `const Foo&` and `Foo const&` equivalent?